sdk/cosmosdb/cosmos/package.json

```diff
@@ -94,6 +94,7 @@
     "@azure/core-auth": "^1.3.0",
     "@azure/core-rest-pipeline": "^1.2.0",
     "@azure/core-tracing": "^1.0.0",
+    "@azure/identity": "^2.0.1",
```

Comment:
Why new dependency for @azure/identity needed?

Is it a prior regression? or new regression?

Reply:
I can use the default logger and revert this file. I will address @azure/identity in the AAD PR.